cloud/src/meta-service/meta_service.cpp

```diff
@@ -1640,8 +1640,8 @@ void MetaServiceImpl::get_tablet_stats(::google::protobuf::RpcController* contro
 
 static bool check_delete_bitmap_lock(MetaServiceCode& code, std::string& msg, std::stringstream& ss,
                                      std::unique_ptr<Transaction>& txn, std::string& instance_id,
-                                     int64_t table_id, int64_t lock_id, int64_t lock_initiator) {
-    std::string lock_key = meta_delete_bitmap_update_lock_key({instance_id, table_id, -1});
```

Comment:
why change this, the key seems same for all method

Reply:
now just using table_id as look_key, but in this pr https://github.com/apache/doris/pull/40431, it will use both table_id and tablet_id as lock_key, so this change will be work in next new future(i.e. https://github.com/apache/doris/pull/40431)